packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -195,10 +195,14 @@ export async function beaconHandlerInit(args: BeaconArgs & GlobalArgs) {
   if (args.private) {
     beaconNodeOptions.set({network: {private: true}});
   } else {
+    const versionStr = `Lodestar/${version}`;
+    const simpleVersionStr = version.split("/")[0];
     // Add simple version string for libp2p agent version
-    beaconNodeOptions.set({network: {version: version.split("/")[0]}});
+    beaconNodeOptions.set({network: {version: simpleVersionStr}});
     // Add User-Agent header to all builder requests
-    beaconNodeOptions.set({executionBuilder: {userAgent: `Lodestar/${version}`}});
+    beaconNodeOptions.set({executionBuilder: {userAgent: versionStr}});
+    // Set jwt version with version string
+    beaconNodeOptions.set({executionEngine: {jwtVersion: versionStr}, eth1: {jwtVersion: versionStr}});
```

Comment:
Maybe should update the description of `--private` flag and mention this as well

https://github.com/ChainSafe/lodestar/blob/18dc0c958189cb60fd8df7db6b1ebdf060edbd96/packages/cli/src/cmds/beacon/options.ts#L119-L120

I guess at some point need to have more general description if there are more things we omit if node is set to private